docs/rp2/quickref.rst

```diff
@@ -112,14 +140,18 @@ Use the ``machine.PWM`` class::
 ADC (analog to digital conversion)
 ----------------------------------
 
-How does the ADC module work?
+RP2040 has five ADC channels in total, four of which are 12-bit SAR based ADCs:
+GP26, GP27, GP28 and GP29. The input signal for ADC0, ADC1 and ADC2 can be
+connected with GP26, GP27, GP28. The fourth is used to measure the VSYS voltage on
+the board. The standard ADC range is 0-3.3V. The fifth channel is connected to
+the in-built temperature sensor and can be used for measuring the temperature by
+reading the analog values.
```

Comment:
I couldn't find anything about VSYS on the 4th channel in the datasheet.  As far as I understand, channels ADC0-ADC3 can be connected with GP26-GP29 (inclusive) and the 5th channel is for temperature.